pkgs/development/python-modules/pylette/default.nix

```
@@ -0,0 +1,67 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  poetry-core,
+  scikit-learn,
+  typer,
+  requests,
+  pillow,
+  numpy,
+  pytestCheckHook,
+  opencv-python,
+  requests-mock,
+}:
+buildPythonPackage rec {
+  pname = "pylette";
+  version = "4.0.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "qTipTip";
+    repo = "Pylette";
+    tag = version;
+    hash = "sha256-i8+QQpYoRfgoV9Nm/FvXSJV+TEvmaaPsPIeG+PU838Q=";
+  };
+
+  build-system = [
+    poetry-core
+  ];
+
+  dependencies = [
+    scikit-learn
+    pillow
+    requests
+    typer
+    numpy
+  ];
+
+  pythonImportsCheck = [
+    "Pylette"
+  ];
+
+  pythonRelaxDeps = [
+    "numpy"
+    "Pillow"
```

```suggestion
    "Pillow"
    "typer"
```

```
Checking runtime dependencies for pylette-4.0.0-py3-none-any.whl
       >   - typer<0.13.0,>=0.12.5 not satisfied by version 0.15.1
```
Did not check whether it would still work.